packages/theme/src/alert/responsive.less

```
@@ -0,0 +1,20 @@
+@import '../custom.less';
+@import './vars.less';
+
+@alert-prefix-cls: ~'@{css-prefix}alert';
+
+@media screen and (max-width: 1280px) {
+  .@{alert-prefix-cls} {
```

Stylelint (17.4.0)

[error] Unknown word alert-prefix-cls (CssSyntaxError)

(CssSyntaxError)